### DISCUSSION TITLE
Ошибка «Схема не найдена» после установки totum из sh-скрипта

### DISCUSSION THREAD

**Discussion by Andremgus** [totumonline/totum-ru-issues-and-discussions#287] [ОШБК!] [answered]
![image](https://user-images.githubusercontent.com/92263004/206698127-90c54d88-9f3f-4dfb-a4b2-f79567297da4.png)

Приветствую. Подскажите что эта за ошибка. Устанавливал totum из скрипта без домена на чистой Ubuntu 20.04 Focal Fossa. Использовал autoinstall_without_domain.sh

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Вы когда устанавливали со свистом пролетели большой красный пункт об определениии ip-адреса сервера. Обычно это бывает, когда ставят на виртуалках развернутых на домашнем ПК (тк там интерфейс определяемый изнутри и снаружи разный).

Вам надо открыть и отредактировать:

```
nano /home/totum/totum-mit/Conf.php
```

Найти там:

```php
/***getSchemas***/
static function getSchemas()
{
return array (
'YOUR_AUTOMATIC_DETECTED_IP_OR_URL' => 'totum',
);}
/***getSchemasEnd***/

```

 И замените `YOUR_AUTOMATIC_DETECTED_IP_OR_URL` на тот ip по которому идет доступ — `192.168....`

**Reply by Andremgus** [reply]
Огонь. Спасибо !!! Заработало без ребута виртуалки ))

**Reply by ttm-alexeypolunin** [reply]
Угу. Conf.php читается при каждом действии в Totum (на самом деле он в в JIT хранится, но все-равно исполняется). Поэтому если вдруг нужно несколько тысяч схем на одном сервера (как наш демо), то это место надо модифицировать, что бы хранить соответствие схем и хостов где-то в БД.
